sdcm/nemesis.py

```diff
@@ -3585,11 +3584,7 @@ def remove_node():
             # In order to eliminate cases that we miss a "data loss" bug because of it, we cleanup this data.
             # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
             try:
-                test_keyspaces = self.cluster.get_test_keyspaces()
-                for node in self.cluster.nodes:
-                    with adaptive_timeout(Operations.CLEANUP, node, timeout=HOUR_IN_SEC * 48):
-                        for keyspace in test_keyspaces:
-                            node.run_nodetool(sub_cmd='cleanup', args=keyspace, retry=0)
+                self.nodetool_cleanup_on_all_nodes_parallel()
```

Comment:
It stopped using keyspace argument, was it done intentionally?

Reply:
yes,
specifying specific keyspaces complicates the work, and might fail on mismatch in schema (i.e. parallel nemesis)

anyhow to instruction for for those procedure is doing full cleanup after topology change (also for system keyspaces)

this change happen more than a year ago, and one might find the whole discussion there.
this PR just port this change into the other relevant places that were forgotten